### PR TITLE
test: EntityManager.insert() with explicit primary keys (#7094)

### PR DESCRIPTION
**Pull request by AdrianDC**
### Pull-Request Checklist

- [x] Code is up-to-date with the `master` branch
- [x] `npm run lint` passes with this change
- [ ] `npm run test` passes with this change
- [x] This pull request links relevant issues as `Fixes #0000`
- [x] There are new or updated unit tests validating the change
- [ ] Documentation has been updated to reflect this change
- [x] The new commits follow conventions explained in [CONTRIBUTING.md](https://github.com/typeorm/typeorm/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -111,7 +112,8 @@
     "sqlite3": "^5.0.0",
     "ts-node": "^9.0.0",
     "typeorm-aurora-data-api-driver": "^1.4.0",
-    "typescript": "~3.6.0"
+    "typescript": "~3.6.0",
+    "uuid": "^8.3.1"
```
Comment (nebkat) [inline package.json:116]: Should use the internal uuid generation:

https://github.com/typeorm/typeorm/blob/c4a36da62593469436b074873eba186f0f8b990d/src/util/RandomGenerator.ts#L149

**Review by nebkat — CHANGES_REQUESTED** [review]

File: test/github-issues/7094/issue-7094.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        console.log(tests);
+
+        expect(tests[0].id).to.equal(uuid1);
+        expect(tests[1].id).to.equal(uuid2);
+        expect(tests[2].id).to.equal(uuid3);
+        expect(tests[3].id).to.equal(uuid4);
+        assert.isNotNull(tests[4].id);
+
+        const items = await repo.find();
+        console.log(items);
```
Comment (imnotjames) [inline test/github-issues/7094/issue-7094.ts:61]: Can you remove this `console.log`

**Review by imnotjames — APPROVED** [review]

**Comment by imnotjames** [comment]
This is still a draft - are you not looking to have it merged?

**Comment by AdrianDC** [comment]
The original issue is detailed on issue https://github.com/typeorm/typeorm/issues/7094.

This PR is meant to reproduce the issue, but as I stated there, I could not get it reproduced back then,
unlike the same code being used in production since version "0.2.26".

**Comment by imnotjames** [comment]
So it doesn't replicate the issue - should we close it instead?

**Pull request closed without merging**